app/features/accounts/account-hooks.ts

```diff
@@ -391,7 +391,23 @@ export function useDefaultAccount() {
   );
 
   if (!defaultAccount) {
-    throw new Error(`No default account found for currency ${defaultCurrency}`);
+    const fallbackAccount = accounts.find(
+      (x) => x.currency === defaultCurrency,
+    );
+    if (!fallbackAccount) {
+      throw new Error(
+        `No default account found for currency ${defaultCurrency}`,
+        {
+          cause: {
```

Comment:
we normally use cause when we wrap original error with a custom wrapper and put other stuff directly into error object